src/test/java/co/cask/cdc/plugins/sink/CDCBigTableConfigTest.java

```diff
@@ -0,0 +1,134 @@
+package co.cask.cdc.plugins.sink;
+
+import co.cask.cdap.etl.api.validation.InvalidConfigPropertyException;
+import com.google.bigtable.repackaged.com.google.cloud.ServiceOptions;
+import org.assertj.core.api.Assertions;
+import org.junit.Assume;
+import org.junit.Test;
+
+public class CDCBigTableConfigTest {
+  private static final String VALID_REF = "test-ref";
+  private static final String VALID_PROJECT = "test-project";
+  private static final String VALID_INSTANCE = "test-instance";
+  private static final String VALID_ACCOUNT_FILE_PATH
+    = CDCBigTableConfigTest.class.getResource("/credentials.json").getPath();
+
+  @Test
+  public void testValidateValidConfig() {
+    CDCBigTableConfig config = new CDCBigTableConfig(
+      VALID_REF,
+      VALID_INSTANCE,
+      VALID_PROJECT,
+      VALID_ACCOUNT_FILE_PATH
+    );
+
+    Assertions.assertThatCode(config::validate)
```

Comment:
This can be replaced with just a call to config.validate().

Reply:
fixed

PS: I prefer BDD in cases where we want to make an accent on what is actually tested here. I understand your point and will simplify such tests in future.